### DISCUSSION TITLE
I can not find code examples in next.js v15 wikis,where can i find some?

### DISCUSSION THREAD

**Discussion by nicevlhjfihtqngkldl** [vercel/next.js#69276] [App Router]
https://nextjs.org/docs/app/building-your-application/rendering/server-components

There are not code examples in this page for [Static Rendering](https://nextjs.org/docs/app/building-your-application/rendering/server-components#static-rendering-default), [Dynamic Rendering](https://nextjs.org/docs/app/building-your-application/rendering/server-components#dynamic-rendering),[Streaming](https://nextjs.org/docs/app/building-your-application/rendering/server-components#streaming)

Could anybody help me find the latest code examples for nextjs v15?
Thanks for your help.

**Comment by ZoranRavic** [comment]
Look for examples for nextjs 14 or 13. 

Nextjs 15 is not released yet.

All of the old examples for the app router should still work for next 15.
